src/TRestRawToDetectorSignalProcess.cxx

```diff
@@ -167,20 +124,11 @@ TRestEvent* TRestRawSignalToSignalProcess::ProcessEvent(TRestEvent* evInput) {
         TRestRawSignal* rawSgnl = fInputSignalEvent->GetSignal(n);
         sgnl.SetID(rawSgnl->GetID());
         for (int p = 0; p < rawSgnl->GetNumberOfPoints(); p++)
-            sgnl.NewPoint(fTriggerStarts + fSampling * p, fGain * rawSgnl->GetData(p));
+            if (rawSgnl->GetData(p) > fThreshold)
```

Comment:
This commit breaks the novel "intwindow" analysis, I would suggest to set to zero in case is below threshold

Reply:
No, why to set to zero, we want to save storage space. It is probably something that needs to be fixed at the `intwindow` method. You probably need to define an integration window.

See for example how it is done at:

https://sultan.unizar.es/rest/classTRestDetectorSignalToRawSignalProcess.html